Add short installation/configuration section to `README.md`

lunkwill42 opened the pull request:
This documents how to install both from PyPI and from source.  **NOTE** however, Zino is not yet published on PyPI, but the guide should work once it's published.

Closes #156

johannaengland commented on polldevs.cf.example:
```diff
@@ -0,0 +1,34 @@
+# defaults, or a new, distinct router.  Router names can be arbitrary, but must
+# be unique across all of Zino.  Comments begin with the # character.
+
+# Default values that will apply to all router entries unless overridden:
+default interval: 5
+default community: public
+default domain: example.org
+default hcounters: yes
```
Would be nice to have more information about those default values (specifically interval and hcounters)

hmpf replied:
What is the time unit for interval: seconds? minutes?

lunkwill42 replied:
minutes, yeah. It says so further down in the file, when `interval` is specified again, but I've added a comment to the example.

johannaengland commented on README.md:
```diff
@@ -12,7 +12,74 @@ replacement for the Tcl-based Zino.
+
+### Running Zino for the first time
+
+You should be able to run Zino (although you will need to configure a list of
+devices to poll in `polldevs.cf` first) from the command line as long as its
+virtual environment is activated:
```
Or in some other way point towards the section about `polldevs.cf`

lunkwill42 replied:
Then the whole example needs to change as well, since it demonstrates Zino's error message when there is no `polldevs.cf` present.

lunkwill42 commented:
Since we added the small configuration section, I realized we should also document how to configure user accounts for the API, so I've added yet another section for that.  Feel free to have another look :)
